kevm-pyk/src/kevm_pyk/utils.py

```diff
@@ -49,3 +103,35 @@ def abstract_cell_vars(cterm: KInner, keep_vars: Collection[KVariable] = ()) ->
         if type(subst[s]) is KVariable and not is_anon_var(subst[s]) and subst[s] not in keep_vars:
             subst[s] = abstract_term_safely(KVariable('_'), base_name=s)
     return substitute(config, subst)
+
+
+def sanitize_config(defn: KDefinition, init_term: KInner) -> KInner:
```

Comment:
Let's create an issue to mark that there is duplicated functionality between `kevm_pyk` and `ksummarize` that eventually should be factored out.

Reply:
I don't think this duplicated functionality will be long-lived. Plan to remove calls to sanitize_config once we have `execute` endpoint working (in my local branch, but not pushed up yet).